### PR TITLE
feat(cd): add rhel-9 matrix

### PR DESCRIPTION
**Pull request by AndyZhang0707**
### Summary

Add rhel 9 matrix

kong/kong-ee#5877

### Checklist

- [ ] The Pull Request has tests
- [ ] There's an entry in the CHANGELOG
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Full changelog

* [Implement ...]

### Issue reference

[KAG-1969](https://konghq.atlassian.net/browse/KAG-1969)

[KAG-1969]: https://konghq.atlassian.net/browse/KAG-1969?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/matrix-commitly.yml [hunk 1]
```diff
@@ -4,21 +4,35 @@ build-packages:
   os: ubuntu-22.04
   package: deb
   check-manifest-suite: ubuntu-22.04-amd64
+- label: rhel-9
```
Comment (fffonion) [inline .github/matrix-commitly.yml:7]: let's revert the changes in commitly.yml before merge

Comment (AndyZhang0707) [reply .github/matrix-commitly.yml]: Reverted as the test has passed.

**Comment by fffonion** [comment]
https://github.com/Kong/kong/actions/runs/5485257938

**Review by fffonion — APPROVED** [review]
